---
layout: default
title: Sun Lab
subtitle: Huiyan Sun Lab at Jilin University
use-site-title: true
---

<div class="container">
    <div class="row">
        <div id="carouselExampleIndicators" class="carousel slide carousel-fade" data-ride="carousel">
          <div class="carousel-inner">
            <div class="carousel-item active" style="height: 31rem">
              <img src="/assets/img/Sun_Group_Photo.jpg" style="position: relative; top: 0px;" class="d-block w-100">
            </div>
          </div>
        </div>
    </div>

    <div class="row" style="margin-top: 1rem">
        <h2>Research Interests</h2>
        <p>Huiyan Sun is an associate professor in Jilin University. Her research interest is cancer systems biology from a causal perspective. Research Keywords: Cancer Systems Biology, Bioinformatics, Causal Learning, Machine Learning, and Large Language Model</p>
    </div>

    <div class="row">
        <h3>Intelligent Algorithms for Enhancing, Generating, and Functionally Analyzing Single-Cell Spatial Multi-Omics</h3>
        <div class="row">
            <div class="col-lg-4">
                <img style="margin-top: 1.5rem;" src="/assets/img/research/STRIDE_Cellist.png">
            </div>
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">Single-cell and spatial multi-omic technologies have revolutionized our understanding of cellular heterogeneity in complex biological systems. However, corresponding analyses currently face various challenges, including inadequate resolution, coverage, and difficulty integrating and generating heterogeneous multi-modal data. To address these issues, we have developed a series of intelligent algorithms. <a href="https://github.com/wanglabtongji/STRIDE">STRIDE</a> and <a href="https://github.com/wanglabtongji/Cellist">Cellist</a> enhance low-resolution and high-resolution spatial transcriptomics data signals, respectively, elevating them to the single-cell level. <a href="https://github.com/wanglabtongji/SCRIP">SCRIP</a> and <a href="https://github.com/wanglabtongji/SCRIPro">SCRIPro</a> constructs gene regulation networks using single-cell and spatial multiomics data. <a href="https://github.com/wanglabtongji/CCI">EvaCCI</a> assesses intercellular interactions , while <a href="https://github.com/wanglabtongji/SCREE">SCREE</a> analyzes multimodal single-cell CRISPR screen data. These algorithms enhance the usability of single-cell spatial omics data and lay the foundation for the integrative analysis of spatiotemporal multimodal data, as well as computational modeling of cellular phenotypes in physiological and pathological conditions.
                </p>
            </div>
        </div>
    </div>

    <div class="row">
        <h3>Cell Phenotype Modeling Based on Generative AI and Large-Scale Spatiotemporal Multimodal Data</h3>
        <div class="row">
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">Cell phenotypes in multicellular systems are regulated by intrinsic factors (such as gene expression regulation) and extrinsic factors (such as intercellular interactions). We have demonstrated the tight connection between intrinsic epigenetic regulations and cell fate determination in mouse IVF and SCNT embryo development (<b>Nature, 2016; Nat. Cell Biol., 2018; Cell Stem Cell, 2018, 2022; Cell Res., 2022</b>). Currently, we are developing generative AI models that are pretrained on large-scale single-cell multi-modal datasets. These models aim to uncover the collaborative effects between gene regulation, cellular crosstalk, and other environmental factors, such as metabolites and mechanics, on cell phenotypes. As a pioneering work, we have developed <a href="https://github.com/wanglabtongji/SELINA.py">SELINA</a>, which utilizes a multi-adversarial domain adaptation network to automatically annotate cell types using a large-scale pretrained human scRNA-seq reference. We hope to leverage generative AI models to gain deeper insights into the molecular mechanisms driving cell phenotypes and further guide and reshape this transformation process.
                </p>
            </div>
            <div class="col-lg-4">
                <img style="margin-top: 1.5rem;" src="/assets/img/research/SELINA.png">
            </div>
        </div>
    </div>

    <div class="row">
        <a href="https://doi.org/10.1158/0008-5472.CAN-19-3392">
            <h3>Metabolic Reprogramming in Cancer Is Induced to Increase Proton Production</h3>
        </a>
        <div class="row">
            <div class="col-lg-4">
                <a href="https://doi.org/10.1158/0008-5472.CAN-19-3392">
                    <img style="margin-top: 1.5rem;" src="/assets/img/research/Cancer Research (2020) 80(5)：1143–1155.png">
                </a>
            </div>
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">
                    Considerable metabolic reprogramming has been observed in a conserved manner across multiple cancer types, but their true causes remain elusive. We present an analysis of around 50 such reprogrammed metabolisms (RM) including the Warburg effect, nucleotide de novo synthesis, and sialic acid biosynthesis in cancer. Analyses of the biochemical reactions conducted by these RMs, coupled with gene expression data of their catalyzing enzymes, in 7,011 tissues of 14 cancer types, revealed that all RMs produce more H+ than their original metabolisms. These data strongly support a model that these RMs are induced or selected to neutralize a persistent intracellular alkaline stress due to chronic inflammation and local iron overload. To sustain these RMs for survival, cells must find metabolic exits for the nonproton products of these RMs in a continuous manner, some of which pose major challenges, such as nucleotides and sialic acids, because they are electrically charged. This analysis strongly suggests that continuous cell division and other cancerous behaviors are ways for the affected cells to remove such products in a timely and sustained manner. As supporting evidence, this model can offer simple and natural explanations to a range of long-standing open questions in cancer research including the cause of the Warburg effect.
                </p>
            </div>
        </div>
    </div>

</div>
