---
layout: default
title: Sun Lab
subtitle: Huiyan Sun Lab at Jilin University
use-site-title: true
---

<div class="container">
    <div class="row">
        <div id="carouselExampleIndicators" class="carousel slide carousel-fade" data-ride="carousel">
          <div class="carousel-inner">
            <div class="carousel-item active" style="height: 31rem">
                <a href="https://sai.jlu.edu.cn/info/1094/3447.htm">
                    <img src="/assets/img/Sun_Group_Photo.jpg" style="position: relative; top: 0px;" class="d-block w-100">
                </a>
            </div>
          </div>
        </div>
    </div>

    <div class="row" style="margin-top: 1rem">
        <h2>Research Interests</h2>
        <p><a href="https://scholar.google.com/citations?user=A9t1pDwAAAAJ&hl=zh-CN&oi=ao">Huiyan Sun</a> is an associate professor in Jilin University. Her research interest is <strong>cancer systems biology from a causal perspective.</strong> <strong>Research Keywords:</strong> Cancer Systems Biology, Bioinformatics, Causal Learning, Machine Learning, and Large Language Model</p>
    </div>

    <div class="row">
        <a href="https://doi.org/10.1093/bib/bbae721">
            <h3>Identifying Cancer Prognosis Genes Through Causal Learning</h3>
        </a>
        <div class="row">
            <div class="col-lg-4">
                <a href="https://doi.org/10.1093/bib/bbae721">
                    <img style="margin-top: 1.5rem;" src="/assets/img/research/BIB 2025 26(1)：bbae721.png">
                </a>
            </div>
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">
                    Accurate identification of causal genes for cancer prognosis is critical for estimating disease progression and guiding treatment interventions. In this study, we propose <a href="https://doi.org/10.1093/bib/bbae721">CPCG (Cancer Prognosis's Causal Gene)</a>, a two-stage framework identifying gene sets causally associated with patient prognosis across diverse cancer types using transcriptomic data. Initially, an ensemble approach models gene expression's impact on survival with parametric and semiparametric hazard models. Subsequently, an iterative conditional independence test combined with graph pruning is utilized to infer the causal skeleton, thereby pinpointing prognosis-related genes. <strong>Experiments on transcriptomic data from 18 cancer types sourced from The Cancer Genome Atlas Project demonstrate CPCG's effectiveness in predicting prognosis under four evaluation metrics.</strong> Validations on 24 additional datasets covering 12 cancer types from the Gene Expression Omnibus and the Chinese Glioma Genome Atlas Project further demonstrate CPCG's robustness and generalizability. CPCG identifies a concise but reliable set of genes, obviating the need for gene combination enumeration for survival time estimation. These genes are also proved closely linked to crucial biological processes in cancer. Moreover, CPCG constructs a stable causal skeleton and exhibits insensitivity to the order of data shuffling. Overall, CPCG is a powerful tool for extracting cancer prognostic biomarkers, offering interpretability, generalizability, and robustness. CPCG holds promise for facilitating targeted interventions in clinical treatment strategies.
                </p>
            </div>
        </div>
    </div>

    <div class="row">
        <a href="https://doi.org/10.1016/j.compbiomed.2024.108690">
            <h3>Sliver: Unveiling Large Scale Gene Regulatory Networks of Single-Cell Transcriptomic Data Through Causal Structure Learning and Modules Aggregation</h3>
        </a>
        <div class="row">
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">
                    Prevalent Gene Regulatory Network (GRN) construction methods rely on generalized correlation analysis. However, in biological systems, regulation is essentially a causal relationship that cannot be adequately captured solely through correlation. Therefore, <strong>it is more reasonable to infer GRNs from a causal perspective.</strong> Existing causal discovery algorithms typically rely on Directed Acyclic Graphs (DAGs) to model causal relationships, but it often requires traversing the entire network, which result in computational demands skyrocketing as the number of nodes grows and make causal discovery algorithms only suitable for small networks with one or two hundred nodes or fewer. In this study, we propose the <a href="https://doi.org/10.1016/j.compbiomed.2024.108690">SLIVER (cauSaL dIscovery Via dimEnsionality Reduction) algorithm</a> which integrates causal structural equation model and graph decomposition. SLIVER introduces a set of factor nodes, serving as abstractions of different functional modules to integrate the regulatory relationships between genes based on their respective functions or pathways, thus reducing the GRN to the product of two low-dimensional matrices. Subsequently, we employ the structural causal model (SCM) to learn the GRN within the gene node space, enforce the DAG constraint in the low-dimensional space, and guide each factor to aggregate various functions through cosine similarity. We evaluate the performance of the SLIVER algorithm on 12 real single cell transcriptomic datasets, and demonstrate it outperforms other 12 widely used methods both in GRN inference performance and computational resource usage. The analysis of the gene information integrated by factor nodes also demonstrate the biological explanation of factor nodes in GRNs. We apply it to scRNA-seq of Type 2 diabetes mellitus to capture the transcriptional regulatory structural changes of β cells under high insulin demand.
                </p>
            </div>
            <div class="col-lg-4">
                <a href="https://doi.org/10.1016/j.compbiomed.2024.108690">
                    <img style="margin-top: 1.5rem;" src="/assets/img/research/CBM 2024 178：108690.png">
                </a>
            </div>
        </div>
    </div>

    <div class="row">
        <a href="https://doi.org/10.1158/0008-5472.CAN-19-3392">
            <h3>Metabolic Reprogramming in Cancer Is Induced to Increase Proton Production</h3>
        </a>
        <div class="row">
            <div class="col-lg-4">
                <a href="https://doi.org/10.1158/0008-5472.CAN-19-3392">
                    <img style="margin-top: 1.5rem;" src="/assets/img/research/Cancer Research (2020) 80(5)：1143–1155.png">
                </a>
            </div>
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">
                    Considerable metabolic reprogramming has been observed in a conserved manner across multiple cancer types, but their true causes remain elusive. <strong>We present an analysis of around 50 such reprogrammed metabolisms (RMs)</strong> including the Warburg effect, nucleotide de novo synthesis, and sialic acid biosynthesis in cancer. Analyses of the biochemical reactions conducted by these RMs, coupled with gene expression data of their catalyzing enzymes, in 7,011 tissues of 14 cancer types, <strong>revealed that all RMs produce more H+ than their original metabolisms.</strong> These data strongly support a model that these RMs are induced or selected to neutralize a persistent intracellular alkaline stress due to chronic inflammation and local iron overload. To sustain these RMs for survival, cells must find metabolic exits for the nonproton products of these RMs in a continuous manner, some of which pose major challenges, such as nucleotides and sialic acids, because they are electrically charged. This analysis strongly suggests that continuous cell division and other cancerous behaviors are ways for the affected cells to remove such products in a timely and sustained manner. As supporting evidence, <strong>this model can offer simple and natural explanations to a range of long-standing open questions in cancer research including the cause of the Warburg effect.</strong>
                </p>
            </div>
        </div>
    </div>

</div>
